bundles/org.openhab.binding.rotel/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -129,7 +129,7 @@
 	</channel-group-type>
 
 	<channel-type id="source">
-		<item-type>String</item-type>
+		<item-type>Selection</item-type>
```

Comment:
Why changing that? 
Selection is not a valid item type.

Reply:
ok: Selection re-changed to String